cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -6624,3 +6624,64 @@ action.cyences_notable_event_action.attacker_search = | stats count by ip_client
 action.cyences_notable_event_action.products = F5 BIGIP
+
+
+[Cloudflare - Credential Leaked]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 27 * * * *
+description = A Credential leaked events from Cloudflare
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_cloudflare` sourcetype="cloudflare:json" LeakedCredentialCheckResult!="none" \
+| eval cyences_severity = "critical" \
+| table _time cyences_severity Kind LeakedCredentialCheckResult Action ClientCountry Description Metadata* Edge* Source Client* \
+| `cs_cloudflare_credential_leaked_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_cloudflare_credential_leaked_filter
+action.cyences_notable_event_action.contributing_events = `cs_cloudflare` sourcetype="cloudflare:json" LeakedCredentialCheckResult!="none" 
+action.cyences_notable_event_action.attacker_search = | stats count by ClientIP
+action.cyences_notable_event_action.attacker_drilldown =  `cs_cloudflare` sourcetype="cloudflare:json" LeakedCredentialCheckResult!="none" ClientIP=$row.ClientIP$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Cloudflare
+
+[Cloudflare - BOT Traffic]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 34 * * * *
+description = A Bot traffic events from Cloudflare
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_cloudflare` sourcetype="cloudflare:json" "Metadata.js_detection"=FAILED \
+| eval cyences_severity = if(Action="block", "low", "medium") \
+| table _time cyences_severity Kind LeakedCredentialCheckResult Action ClientCountry Description Metadata* Edge* Source Client* \
+| `cs_cloudflare_bot_traffic_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_cloudflare_bot_traffic_filter
+action.cyences_notable_event_action.contributing_events =  `cs_cloudflare` sourcetype="cloudflare:json" "Metadata.js_detection"=FAILED
+action.cyences_notable_event_action.attacker_search = | stats count by ClientIP
+action.cyences_notable_event_action.attacker_drilldown = `cs_cloudflare` sourcetype="cloudflare:json" "Metadata.js_detection"=FAILED ClientIP=$row.ClientIP$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Cloudflare
```

Comment:
Add team category for this alert:

action.cyences_notable_event_action.teams = **SOC or Compliance**